README.md

```diff
@@ -255,3 +245,56 @@ You may run multiple controls:
 bundle
 rake test:integration[azure_resource_group,azurerm_virtual_machine]
 ```
+
+### Optional Components
+
+By default, rake tasks will only use core components. Optional components have
+associated integrations that will be skipped unless you enable these. We have
+the following optional pieces that may be managed with Terraform.
+
+Optional Components may be combined when running tasks:
+
+```
+rake options[option_1,option_2,option3]
+direnv allow # or source .envrc
+rake tf:apply
+```
+
+To disable optional components just re-run `rake options` without the components you wish to disable:
```

Comment:
Reword:

To disable optional components run `rake options` including only the optional components you wish to enable. Any omitted component will be disabled.